src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -290,6 +292,7 @@ def encode_prompt(
         prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_prompt_embeds: Optional[torch.FloatTensor] = None,
         lora_scale: Optional[float] = None,
+        **kwargs,
```

Comment:
```suggestion
        clip_skip: Optional[int] = None,
```

Reply:
**kwargs should be added to deprecated functions but not to official, public API functions

Reply:
+ we should add docs